inspirehep/modules/disambiguation/search.py

```diff
@@ -0,0 +1,367 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Set of methods to query Elasticsearch instance."""
+
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+)
+
+from dateutil.parser import parse
+
+from elasticsearch.helpers import scan
+from flask import current_app
+
+from invenio_search import current_search_client as es
+
+
+def _get_record(record_id, source):
+    """Make a call to the Elasticsearch instance with known record_id.
+
+    :param record_id:
+        An identification (UUID) of the given record.
+
+        Example:
+            record_id = "a5afb151-8f75-4e91-8dc1-05e7e8e8c0b8"
+
+    :param source:
+        A list of fields to be extracted from the Elasticsearch response.
+
+        Example:
+            source = ["collaboration", "author.full_name"]
+    """
+    elasticsearch_index = current_app.config.get('DISAMBIGUATION_RECORD_INDEX')
```

Comment:
As I said today during standup, record search MUST be centralized, otherwise we would have to change the index name everywhere it's used when we deploy on production.